Tests/scripts/gitlab_slack_notifier.py

```diff
@@ -49,9 +49,10 @@ def options_handler():
 
 
 def unit_tests_results():
-    failing_unit_tests = get_failing_unit_tests_file_data()
+    failing_unit_tests = get_artifact_data('failed_lint_report.txt')
```

Comment:
why do we have this in gitlab build if we don't run unit tests there?

Reply:
We do run unit tests there in various flows.